libs/arcade-cli/arcade_cli/templates/{{ toolkit_name }}/Makefile

```diff
@@ -8,14 +8,14 @@ help:
 install: ## Install the uv environment and install all packages with dependencies
 	@echo "🚀 Creating virtual environment and installing all packages using uv"
 	@uv sync --active --all-extras --no-sources
-	@uv run pre-commit install
+	@if [ -f .pre-commit-config.yaml ]; then uv run pre-commit install; fi
```

Comment:
I think you're missing the --no-sources flag? 

`@if [ -f .pre-commit-config.yaml ]; then uv run --no-sources pre-commit-install; fi`